propertyRating/app/views/properties/_form.html.erb

```diff
@@ -1,12 +1,9 @@
 <%= form_for(@property) do |f| %>
   <% if @property.errors.any? %>
     <div id="error_explanation">
-      <h2><%= pluralize(@property.errors.count, "error") %> prohibited this property from being saved:</h2>
-
+      <h2>Property already exists in database</h2>
```

Comment:
This isn't expandable at all. You are saying that if there are any errors, display "Property already exists."

What if you add other validations?

When you use a validation, the validation has its own error messages. They get pushed into @property.errors

See https://github.com/bhogancvtc/rails_blog_example/blob/master/app/views/admin/posts/_form.html.erb#L16-L18